bobikApp/TableVC.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  TableVC.swift
+//  bobikApp
+//
+//  Created by Vasyl Denys on 1/21/24.
+//
+
+import UIKit
+
+class TableVC: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        for i in 1...5 {
+            let man = Person(name: "Bob \(i)")
+            people.append(man)
+        }
+        
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = 100
+    }
+    
+    var people = [Person]()
```

Comment:
Try filling this array with data right there. You can make this property lazy.

Reply:
done